scripts/Microsoft.DotNet.Cli.Build.Framework/TargetConditions/EnvironmentAttribute.cs

```diff
@@ -24,15 +23,22 @@ public override bool EvaluateCondition()
         {
             var actualVal = Environment.GetEnvironmentVariable(_envVar);
 
-            foreach (var expectedVal in _expectedVals)
+            if (_expectedVals.Length == 0)
             {
-                if (string.Equals(actualVal, expectedVal, StringComparison.Ordinal))
+                return !string.IsNullOrEmpty(actualVal);
+            }
+            else
```

Comment:
I considered that, but I think using an `if/else` instead of just an `if` makes the logic clearer. To me, at least, removing the `else` would make the code read like the `_expectedVals.Length == 0` case is an edge case or error condition, whereas I want it to convey that these are just two different options for the use of the method.

Also because the `else` block is small, I'd prefer to keep it as is, because I think the logical clarity outweighs the nesting in this instance.